### DISCUSSION TITLE
how can doris routine load with kafka sasl_ssl authentication work?

### DISCUSSION THREAD

**Discussion by liuxiahuiyi** [apache/doris#7861] [A - General / Q&A]
I create a doris routine load, but the kafka source requires sasl_ssl authentication. Then I add config in routine load like  

FROM KAFKA (
  "kafka_broker_list" = "",
  "kafka_topic" = "",
  "property.kafka_default_offset" = "OFFSET_END",
  "property.security.protocol" = "SASL_SSL",
  "property.sasl.mechanism" = "SCRAM-SHA-512",
  "property.sasl.username" = "",
  "property.sasl.password" = "",
  "property.ssl.ca.location" = "FILE:ca_dev.pem"
)
But the routine load job can not authenticate the kafka, anything else I should add?

**Comment by liuxiahuiyi** [comment]
The doris version is 0.15.0

**Comment by dataalive** [comment]
问题解决了吗？

**Comment by morningman** [comment]
It has been supported in #7251, and will be available in Doris v1.0(coming soon)
